README.md

```diff
@@ -30,58 +30,42 @@ Discord bot for Eddie Jaoude's Discord server
 
 - GitHub Actions deploys mainline branch to Azure
 
-## Required
+## → Requirements
 
-- node v10+ installed
+- Docker and Docker-Compose
 - discord token. To get one follow [these instructions](https://discordjs.guide/preparations/setting-up-a-bot-application.html#creating-your-bot)
 - general channel ID of your discord server, read the instructions on one of these links to get yours:
    - [Get the channel ID of the Discord text channel](https://github.com/Chikachi/DiscordIntegration/wiki/How-to-get-a-token-and-channel-ID-for-Discord#get-the-channel-id-of-the-discord-text-channel)
    - [Where can I find my User/Server/Message ID?](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
 - ID of your discord server, to get your server ID follow these steps:
    - First make sure you have **Developer Mode** enabled on your Discord by visiting your Discord settings and going to Appearance
-
-   ![user settings button](https://user-images.githubusercontent.com/18630253/83634306-3458c180-a59a-11ea-8a96-15e9751b9d08.png)
-   ![developer mode toggle button](https://user-images.githubusercontent.com/18630253/83634441-7a158a00-a59a-11ea-919c-2d7384d724f7.png)
-
    - Right click on the server icon on the left hand sidebar then click on "Copy ID"
 - firebase key for the project, check [these docs](https://firebase.google.com/docs/admin/setup) to get your key
 - [optional] GCP account to deploy to (using GitHub Actions)
 
-## Quickstart
-
-1. Clone the project or your fork (if you plan to make changes use your fork)
-2. Install dependencies by running the command `npm install`
+## → How to run EddieBot locally:
 
-### Run the project locally on Mac and Linux using below command
-1. `DISCORD_TOKEN=<GET YOUR DISCORD TOKEN> GENERAL_CHANNEL_ID=<GET YOUR GENERAL CHANNEL ID> DISCORD_SERVER_ID=<YOUR SERVER_ID> DISCORD_BOT_CHANNEL_ID=<ID> COMMAND_PREFIX=<PREFIX> FIREBASE_KEY=<FIREBASE_KEY> npm run start:local`
+#### 1. Set Environment Variables
+1. Copy `.env.example` to `.env`.
+1. Generate the "Service Account" from Firebase Settings > Cloud Messaging.
+    1. Download Service Account JSON file from this same screen.
+1. Open `.env` and fill empty strings with matching credentials from the JSON file.
```

Comment:
lots of `1`s here 😜

Reply:
<img width="1443" alt="image" src="https://user-images.githubusercontent.com/150512/93220486-83e78580-f764-11ea-97c4-88582b34cbbc.png">

Reply:
Fancy 👍